pyproject.toml

```diff
@@ -1,18 +1,19 @@
 [tool.poetry]
 name = "pySigma-backend-loki"
-version = "0.9.1"
+version = "0.10.0"
 description = "pySigma Loki backend"
 readme = "README.md"
-authors = ["Nick Moore <nicholas.moore@grafana.com>", "Mostafa Moradian <mostafa.moradian@grafana.com>"]
+authors = [
+    "Nick Moore <nicholas.moore@grafana.com>",
+    "Mostafa Moradian <mostafa.moradian@grafana.com>",
+]
 license = "AGPL-3.0-only"
 repository = "https://github.com/grafana/pySigma-backend-loki"
-packages = [
-    { include = "sigma" }
-]
+packages = [{ include = "sigma" }]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-pysigma = "^0.9.11"
+pysigma = "^0.10.4"
 pysigma-pipeline-sysmon = "^1.0.1"

```

Comment:
Given pysigma-pipeline-sysmon 1.0.1 still depends on pySigma 0.9, I would suggest updating it to 1.0.3 also.

Reply:
It's already up to date in the poetry lock file, but I'll change this file as well.

Reply:
Done! ✅